conf/application.conf

```diff
@@ -157,33 +163,43 @@ hec-tax-check {
 hec-file-extraction-details {
```

Comment:
just to double check, did you check if the app-config's needed updating as well? It'd be worth raising PR's for those at the same time as this if they need updating

Reply:
yes, i am updating that only

Reply:
PR sent